src/server/profiles/metadata/metadata.interactions.js

```diff
@@ -32,6 +32,10 @@ let generateInteractions = (service, resourceType) => {
 		interactions.push({ code: 'delete' });
 	}
 
+	if (service.searchByHistoryVersionId) {
```

Comment:
can we rename this to searchByVersionId?

Reply:
ok